co-authors-plus.php

```diff
@@ -1644,6 +1644,33 @@ public function filter_author_archive_title( $title ) {
 		$author = $this->get_coauthor_by( 'user_nicename', $author_slug );
 		
 		return sprintf( __( 'Author: %s' ), $author->display_name );
+	}	
+
+	/**
+	 * Get the post count for the guest author
+	 *
+	 * @param object $guest_author guest-author object.
+	 * @return int post count for the guest author
+	 */
+	public function get_guest_author_post_count( $guest_author ) {
+		if ( ! is_object( $guest_author ) ) {
+			return;
+		}
+
+		$term = $this->get_author_term( $guest_author );
+		$guest_term = get_term_by( 'slug', 'cap-' . $guest_author->user_nicename, $this->coauthor_taxonomy );
+
+		if ( ! $term || ! $guest_term ) {
```

Comment:
I'm not sure if this check is necessary since it will return 0 in line 1672 if either conditions are met. What are your thoughts?

Reply:
I think it is, since if either of those values is `false` like you mentioned in a previous comment, it would throw a notice - `Trying to get property of non-object`

Reply:
do you think this check makes the last condition on lines 1671 to 1673 redundant? I think it'd be safe to remove those three lines instead of removing this check

Reply:
What if `$guest_term` returns false but `$term` returns a valid value?

Reply:
That's correct, that check is still necessary.